bin/diagnostics.bash

```diff
@@ -7,32 +7,38 @@ here="$(dirname "$(readlink -f "$0")")"
 source "${here}/common.bash"
 
 usage() {
-    log_info "usage: ck8s diagnostics <sc|wc> [--namespace namespace] [--include-config]"
+    log_info "usage: ck8s diagnostics <sc|wc> [--namespace namespace] [--include-config] [--with-metrics-since date] [--query-metric metric]"
```

Comment:
Nit
```suggestion
    log_info "usage: ck8s diagnostics <sc|wc> [--namespace namespace] [--include-config] [--with-default-metrics-since date] [--with-metric metric]"
```
or
```suggestion
    log_info "usage: ck8s diagnostics <sc|wc> [--namespace namespace] [--include-config] [--query-default-metrics-since date] [--query-metric metric]"
```
Just from the name I initially thought one was simply time range and the other the metric